pkg/apis/tensorflow/v1/types.go

```diff
@@ -60,6 +60,11 @@ type TFJobSpec struct {
 	// +optional
 	SuccessPolicy *SuccessPolicy `json:"successPolicy,omitempty"`
 
+	// FailurePolicy defines the policy to mark the TFJob as failure.
+	// Default to "", using the default rules.
+	// +optional
+	FailurePolicy *FailurePolicy `json:"FailurePolicy,omitempty"`
```

Comment:
Should we use `json:"failurePolicy,omitempty"` here to be consistent?

Reply:
Thanks @andreyvelich. Good catch. I've updated it.